src/openrct2-ui/windows/GameBottomToolbar.cpp

```diff
@@ -601,8 +601,8 @@ static void window_game_bottom_toolbar_draw_news_item(rct_drawpixelinfo* dpi, rc
                 return;
 
             auto clipCoords = ScreenCoordsXY{ 10, 19 };
-
-            if (peep->Is<Staff>() && peep->AssignedStaffType == StaffType::Entertainer)
+            auto* staff = peep->As<Staff>();
+            if (staff != nullptr && staff->AssignedStaffType == StaffType::Entertainer)
```

Comment:
Since `staff` isn't used after this if, its scope can be reduces by writing it like this:
```suggestion
            if (auto* staff = peep->As<Staff>(); staff != nullptr && staff->AssignedStaffType == StaffType::Entertainer)
```

Though since it does more than just checking for nullptr, it might not be the best use case.

Reply:
I've previously used that style for if's but it was agreed that as its not used in the rest of the codebase it would be best to be consistent and not use it.